Q4.txt

```diff
@@ -0,0 +1,28 @@
+JavaScript:
+
+var a = ["john", "mike", "susan", "matt", "joe", "larry", "ken", "samantha", "jenny", "Sara", "henry"];
+var x=0;
+var isprint=true;
+while (x<a.length){
```

Comment:
it prints the first word right then reverse the first and the second and so on print the third one right and then reverse the third one and fourth one.